### PR TITLE
PCHR-3409: Fix datepicker styling

### PR DESCRIPTION
**Pull request by igorpavlov-zz**
## Overview

The button `height` property was added here https://github.com/compucorp/org.civicrm.shoreditch/commit/678c2660de525f48c68edeb702e31907ba0bb888#diff-dcce3294b88d7236dde50eace88b572bR375 by moving the buttons style to a global scope. This broke the layout in several places.

The `height` property is not needed in buttons styling since the calculated height should be based on `font-size`, `line-height`, `padding` and `border`.

This PR removes the `height` CSS property from buttons styling and thus fixes the issue with datepicker styling.

## Before

image

## After

image

## Comments

This PR was tested in both CiviHR and Vanilla CiviCRM.

Here is a backstop diff with the state **before** regression. Only font-size is changed, which is expected, the buttons look fine.

backst

------
✅Manual Tests - passed
✅Backstop tests (CiviCRM + CiviHR) - passed
✅CSS distributive files - included

### CODE REVIEW

**Comment by mukeshcompucorp** [comment]
@igorpavlov can you confirm that /civicrm/a/#/status page does not have any effects due to this change? or whatever is changed it looking proper?

File: scss/civicrm/common/_buttons.scss [hunk 1]
```diff
@@ -382,7 +383,6 @@ button {
   border-radius: $border-radius-base;
   color: $crm-white;
   font-family: $font-family-base;
-  height: 36px;
```
Comment (mukeshcompucorp) [inline scss/civicrm/common/_buttons.scss:385]: Now that Vanilla was working fine with this change. Can you make sure that any page is not left. 
If backstop is not loading page than check that page manually?
This is very generic and I want be absolutely sure that, we don't break anything due to this

**Review by mukeshcompucorp — CHANGES_REQUESTED** [review]

**Comment by igorpavlov-zz** [comment]
@mukeshcompucorp 
The status page before the change:
image
You may notice the buttons are broken (not only the text colour, but also **height**)

After the change:
image

Buttons have correct height.

File: scss/civicrm/common/_buttons.scss (hunk 1 above)
Comment (igorpavlov-zz) [reply scss/civicrm/common/_buttons.scss:385]: Some backstop pages did not load, I tested them manually. A couple of pages (such as pledge dashboard) do not open for me, but this change does only affects buttons and I tested enough buttons on other pages - they look fine.

P.S. Vanilla was partially broken due to this change, see **Comments** section.

Comment (mukeshcompucorp) [reply scss/civicrm/common/_buttons.scss:385]: OK thanks

**Review by mukeshcompucorp — APPROVED** [review]

**Pull request merged**